<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>These are your assets definition</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView">
     <property name="tabKeyNavigation">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnAddNewAsset">
       <property name="text">
        <string>Issue new asset</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../icons/add.png</normaloff>../icons/add.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAddExistingAsset">
       <property name="text">
        <string>Add existing asset</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../icons/add.png</normaloff>../icons/add.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnImportAssetFromJSON">
       <property name="text">
        <string>Import asset from JSON</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../icons/add.png</normaloff>../icons/add.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionCopyMoniker">
   <property name="text">
    <string>Copy &amp;Moniker</string>
   </property>
   <property name="toolTip">
    <string>Copy Moniker</string>
   </property>
  </action>
  <action name="actionCopyColorSet">
   <property name="text">
    <string>Copy &amp;Color set</string>
   </property>
   <property name="toolTip">
    <string>Copy Color set</string>
   </property>
  </action>
  <action name="actionCopyUnit">
   <property name="text">
    <string>Copy &amp;Unit</string>
   </property>
   <property name="toolTip">
    <string>Copy Unit</string>
   </property>
  </action>
  <action name="actionCopyAsJSON">
   <property name="text">
    <string>Copy As &amp;JSON</string>
   </property>
   <property name="toolTip">
    <string>Copy As JSON</string>
   </property>
  </action>
  <action name="actionShowAddresses">
   <property name="text">
    <string>&amp;Show addresses</string>
   </property>
   <property name="toolTip">
    <string>Show addresses</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
